ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/validation/ValidatedStatement.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.server.validation;
+
+import io.confluent.ksql.statement.ConfiguredStatement;
+
+/**
+ * A class that wraps a {@link ConfiguredStatement} with whether or not
+ * its result should be returned to the user. This class can be removed
+ * when we deprecate {@link io.confluent.ksql.parser.tree.RunScript} in
+ * a future release.
+ */
+public final class ValidatedStatement {
```

Comment:
Immutable types! :D

Reply:
Unfortunately, `ConfiguredStatement` is not immutable and cannot be since the overrides need to change - so I can't make this technically immutable either